components/compliance-service/migrations/migrations.go

```diff
@@ -0,0 +1,112 @@
+package migrations
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/olivere/elastic/v7"
+	"github.com/sirupsen/logrus"
+)
+
+type Node struct {
```

Comment:
We already have a struct node, please use that. Its in es_types.go.

Reply:
The Node struct we have in es_types.go, doeesn't have end_time tag